doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -148,99 +164,127 @@ Values are provided in kilobytes (KB).
-      | :ref:`Thermostat <matter_thermostat_sample>` (Debug + LTO)           |            48 |               703 |              4 |         32 |         787 |                             178 |
+      | :ref:`Thermostat <matter_thermostat_sample>` (Debug + LTO)           |            48 |               710 |              4 |         40 |         802 |                             185 |
       +----------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Release)               |            48 |               667 |              4 |         32 |         751 |                             167 |
+      | :ref:`Thermostat <matter_thermostat_sample>` (Release)               |            48 |               673 |              4 |         40 |         765 |                             176 |
       +----------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Debug)       |            48 |               764 |              4 |         32 |         848 |                             174 |
+      | :ref:`Window Covering <matter_window_covering_sample>` (Debug)       |            48 |               760 |              4 |         40 |         852 |                             184 |
       +----------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Debug + LTO) |            48 |               686 |              4 |         32 |         770 |                             178 |
+      | :ref:`Window Covering <matter_window_covering_sample>` (Debug + LTO) |            48 |               694 |              4 |         40 |         786 |                             185 |
       +----------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Release)     |            48 |               655 |              4 |         32 |         739 |                             167 |
+      | :ref:`Window Covering <matter_window_covering_sample>` (Release)     |            48 |               660 |              4 |         40 |         752 |                             177 |
       +----------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
+   .. tab:: nRF54l15 DK with TFM
```

Comment:
```suggestion
   .. tab:: nRF54L15 DK with TF-M
```

Reply:
I think here the usage of TF-M actually made sense -- it's shorter and gives you immediately the idea of component used. In the description, however, it's better to be consistent with most of the mentions in docs.

Reply:
Changed.